modules/checking-cluster-resources-before-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,195 @@
+[source,terminal]
+----
+NAMESPACE    NAME                      AGE
+<namespace>  <example_egress_policy>   5d
+----
+
+.. You can check the intended behavior of an EgressFirewall resource by passing in the `-o yaml` flag. For example:
++
+[source,terminal]
+----
+$ oc get egressnetworkpolicy <example_egress_policy> -n <namespace> -o yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: network.openshift.io/v1
+kind: EgressNetworkPolicy
+metadata:
+  name: <example_egress_policy>
+  namespace: <namespace>
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: 0.0.0.0/0
+  - type: Deny
+    to:
+      cidrSelector: 10.0.0.0/8
+----
++
+Take note of the intended behavior of this resource. It might change after migration.
+
+. Check your cluster for EgressIP resources:
+
+.. Enter the following command to list namespaces with EgressIP resources:
++
+[source,terminal]
+----
+$ oc get netnamespace -A | awk '$3 != ""'
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME        NETID      EGRESS IPS
+namespace1  14173093   ["10.0.158.173"]
+namespace2  14173020   ["10.0.158.173"]
+----
+
+.. Enter the following command to obtain information about a specific EgressIP object:
++
+[source,terminal]
+----
+$ oc get egressips -n <namespace-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                 EGRESSIPS       ASSIGNED NODE                   ASSIGNED EGRESSIPS
+example-egress-ns    10.10.100.253   ip-10-10-156-122.ec2.internal   10.10.150.253
+example-egress-pod   10.10.100.254   ip-10-10-156-122.ec2.internal   10.10.150.254
+----
+
+.. Enter the following command to provide detailed information about an EgressIP:
++
+[source,terminal]
+----
+$ oc describe egressip <egressip-name> -n <namespace-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         egressip-sample
+Namespace:    
+Labels:       <none>
+Annotations:  <none>
+API Version:  k8s.ovn.org/v1
+Kind:         EgressIP
+Metadata:
+  Creation Timestamp:  2024-07-24T18:29:25Z
+  Generation:          1
+  Resource Version:    386572
+  UID:                 66ad1209-a0b3-452f-a513-280ff69e03fb
+Spec:
+  Egress I Ps:
+    192.0.2.10
+    192.0.2.11
+  Namespace Selector:
+    Match Labels:
+      Name:  my-namespace
+Events:      <none>
+----
++
+Note the intended behavior of this resource, because it might change after migration.
+
+. Check your cluster for multicast resources:
+
+.. Enter the following command to locate namespaces with multicast enabled:
++
+[source,terminal]
+----
+$ oc get netnamespace -o json | jq -r '.items[] | select(.metadata.annotations."netnamespace.network.openshift.io/multicast-enabled" == "true") | .metadata.name'
+----
++
+.Example output
++
+[source,terminal]
+----
+namespace1
+namespace3
+----
+
+.. Enter the following command to obtain information about multicast network policies within a namespace:
++
+[source,terminal]
+----
+$ oc get networkpolicy -n <namespace>
+----
++
+.Example output
```

Comment:
multicast and network policy are different feature. 
what's the mean from line 147 to 195.  I think it can be removed.

Reply:
Removed